core/controlplane/config/templates/cluster.yaml

```diff
@@ -13,6 +13,9 @@ s3URI: {{.S3URI}}
 # The AMI ID of CoreOS.
 amiId: "{{.AmiId}}"
 
+# CoreOS has automatic updates https://coreos.com/os/docs/latest/update-strategies.html. This can be a risk in certain situations. Set this param to true to disable it in all instances.
+disableCoreOSUpdates: false
```

Comment:
I'd prefer `disableContainerLinuxAutomaticUpdates` as it has officially renamed to Container Linux a year or so ago 😃